script/test

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# script/test: Run the test suite for the application. Optionally pass in a path
+#              to an individual test file to run a single test.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+if [ -n "$DEBUG" ]; then
+  set -x
+fi
+
+set +e
+echo "==> Setting up test user..."
+createuser --super test >/dev/null 2>&1
```

Comment:
Should `DATABASE_URL` in `.env.test` be updated to use this `test` user account?